### ISSUE TITLE
Version of spec not prominent

### ISSUE THREAD

**Issue by jayphelps** [json-api/json-api#560]
When looking at the spec, AFAIK it does not actually mention what version you're reading. It only notes it on the [status page](http://jsonapi.org/status/), but that's not ideal because if you're coming back to the spec often you should prominently see the version so you casually are aware whether or not it has changed.

Ideally, the version would be listed in the top nav next to the logo and also reiterated at the top of the specification page.

Links to previous revisions would be ideal, but that's less of an issue until 1.0+

**Comment by hhware** [comment]
At this point, even versioning strategy of the spec is not yet defined, see #432.

**Comment by SFEley** [comment]
:+1:  — this has been a problem on my team.  We all _want_ to follow the JSON API, but it's unclear when it changes, and in review we end up debating whether compliance with version "what I remember reading last Tuesday, no, I'm _sure_ this is what it said" is valid or not.  Tracking back through the Github history of `format/index.md` in this project _sounds_ like it ought to help but in practice the changes there are also pretty hard to follow.

If/when a versioning strategy does exist, it would help to to reflect it in Git tags here.  Meanwhile, at the very least giving a revision date on the page and a bulleted mini-changelog of dates when major changes were made would be very helpful for developers shooting at this moving target.

**Comment by dgeb** [comment]
We will be tracking changes closely after 1.0 is tagged on May 21.

**Comment by dgeb** [comment]
Closed via #727
